Fix:

error: implicit declaration of function 'main_window_add_status_icon' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/Tools/gmpc-database-update-tracker.c.orig	2011-08-16 22:10:15.000000000 -0500
+++ src/Tools/gmpc-database-update-tracker.c	2023-11-16 09:58:27.000000000 -0600
@@ -35,6 +35,7 @@
 #include <plugin.h>
 #include <gmpc-profiles.h>
 #include <main.h>
+#include "status_icon.h"
 
 #define _g_object_unref0(var) ((var == NULL) ? NULL : (var = (g_object_unref (var), NULL)))
 #define _g_free0(var) (var = (g_free (var), NULL))
